### PR TITLE
Added SUB and SBM function

### PR DESCRIPTION
**Pull request by alanabreu106**

### CODE REVIEW

File: js_dev/ibm704sim.js [hunk 1]
```diff
@@ -967,6 +967,10 @@ function ADD(computer, address) {
     computer.accumulator.fixed_point += computer.general_memory[address].fixed_point;
 }
```
Comment (equationcrunchor) [inline js_dev/ibm704sim.js]: Add a doc comment for this function like the ones for the other functions.

File: js_dev/ibm704sim.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // there are no index registers or Type A functions.  Or most of the Type B functions
 // basically most of the computer is missing
 
-const no_to_operation_b = {0o601: STO, 0: HTR, 0o500: CLA, 0o400: ADD};
+const no_to_operation_b = {0o601: STO, 0: HTR, 0o500: CLA, 0o400: ADD, 0o402: SUB, 0o040: SBM}; //Change num later
```
Comment (equationcrunchor) [inline js_dev/ibm704sim.js]: Can you put what the correct number should be in the comment?  Thanks!

File: js_dev/ibm704sim.js [hunk 3]
```diff
@@ -967,6 +967,21 @@ function ADD(computer, address) {
     computer.accumulator.fixed_point += computer.general_memory[address].fixed_point;
 }
 
+/**
+ * Emulates the IMB 704 SUB operation.
+ *
+ * Subtracts the value of the storage register to the accumulator as if it were a
+ * fixed point number.
+ *
+ * @param computer
```
Comment (equationcrunchor) [inline js_dev/ibm704sim.js]: Be sure to provide little descriptions for the parameters like in the other doc comments.

**Comment by mscuthbert** [comment]
Watch line lengths -- wrap lines at 100 characters.  Merging, but would welcome a PR to fix.  Thanks! Great work!

**Pull request merged**